api/v1alpha1/obresourcerescue_types.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	tasktypes "github.com/oceanbase/ob-operator/pkg/task/types"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// OBResourceRescueSpec defines the desired state of OBResourceRescue
+type OBResourceRescueSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	TargetKind    string `json:"targetKind"`
+	TargetResName string `json:"targetResName"`
+	Type          string `json:"type"`
+	TargetGV      string `json:"targetGV,omitempty"`
+	Namespace     string `json:"namespace,omitempty"`
+	TargetStatus  string `json:"targetStatus,omitempty"`
+}
+
+// OBResourceRescueStatus defines the observed state of OBResourceRescue
+type OBResourceRescueStatus struct {
+	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+	OperationContext *tasktypes.OperationContext `json:"operationContext,omitempty"`
```

Comment:
OperationContext is not used in controller

Reply:
OK, I have removed it and update manifests with `make generate` and `make manifests`